pkg/master/master.go

```diff
@@ -384,6 +386,26 @@ func (m *Master) InstallAPIs(c *Config) {
 		apiGroupsInfo = append(apiGroupsInfo, apiGroupInfo)
 	}
 
+	if c.APIResourceConfigSource.AnyResourcesForVersionEnabled(authenticationv1beta1.SchemeGroupVersion) {
+		resources := m.getAuthenticationResources(c)
```

Comment:
> nit: following the convention of the the other scheme groups above and below: authenticationResources

Bad convention.  Also, it all goes away in https://github.com/kubernetes/kubernetes/pull/27554